Overview of ASP.NET Core updates (#35802)
dotnet/AspNetCore.Docs: aspnetcore/includes/benefits.md

ASP.NET Core provides the following benefits:

:::moniker range=">= aspnetcore-6.0"

<!-- AUTHOR NOTE: >=6.0 content showcases Blazor and 
                  Minimal APIs, demotes RP/MVC -->

* A unified story for building web apps, web APIs, Azure IoT (Internet of Things) apps, and mobile backends.
* Architected for testability.

> * [Blazor](xref:blazor/index) lets you create rich interactive client-side UIs using [.NET](/dotnet/standard/tour) and [C#](/dotnet/csharp/) with wide browser support based on HTML/JavaScript, including mobile browsers. You can also build hybrid desktop and mobile apps with .NET and Blazor.

Reviewer: Blazor isn't just for client-side. It does server-side rendering too.

```suggestion
* Create rich interactive web UI with [Blazor](xref:blazor/index) using [C#](/dotnet/csharp/) - no JavaScript required. Reuse your Blazor components to build native mobile and desktop apps with [Blazor Hybrid](xref:blazor/hybrid/index).
```

Author: The style manual will want an em dash there ... `&mdash;` ... &mdash; ... with no spaces. I'll edit your suggestion to apply it.

* [Minimal APIs](xref:fundamentals/minimal-apis) are a simplified approach for building fast web APIs with minimal code and configuration by fluently declaring API routes and actions.

> * Supports [Razor Pages](xref:razor-pages/index) and [Model-View-Controller (MVC)](xref:mvc/overview) app development.

Reviewer: This benefits list is really targeted at new users. I don't want to confuse them by mentioning MVC & Razor Pages here.

```suggestion
```

* Ability to develop and run on Windows, macOS, and Linux.

> * Open-source and [community-focused](https://live.asp.net/).

Reviewer: ```suggestion
* Open-source and [community-focused](https://dotnet.microsoft.com/platform/community).
```

> * Open-source and [community-focused](https://live.asp.net/).

Reviewer: These two bullets seem stuck in the middle the various ASP.NET Core frameworks (Blazor, Minimal APIs, SignalR, gRPC). We move them so that all the frameworks are listed together?

* Integrate seamlessly with popular client-side frameworks and libraries, including [Angular](/visualstudio/javascript/tutorial-asp-net-core-with-angular), [React](/visualstudio/javascript/tutorial-asp-net-core-with-react), [Vue](/visualstudio/javascript/tutorial-asp-net-core-with-vue), and [Bootstrap](https://getbootstrap.com/). 

> * Support for hosting Remote Procedure Call (RPC) services using [gRPC](xref:grpc/index).

Reviewer: We should include something SignalR:

```suggestion
* Add real-time web functionality to apps using [SignalR](xref:signalr/index).
* Support for hosting Remote Procedure Call (RPC) services using [gRPC](xref:grpc/introduction).
```

> * A cloud-ready, environment-based [configuration system](xref:fundamentals/configuration/index).

Reviewer: I think we want to mention something about security as well:

```suggestion
* Built-in security features for [authentication](xref:security/authentication/index) and [authorization](xref:security/authorization/introduction).
* A cloud-ready, environment-based [configuration system](xref:fundamentals/configuration/index).
```
